packages/ui5-task-zipper/lib/zipper.js

```diff
@@ -0,0 +1,31 @@
+
+const yazl = require("yazl");
+const resourceFactory = require("@ui5/fs").resourceFactory;
+const log = require("@ui5/logger").getLogger("builder:customtask:zipper");
+
+module.exports = async function ({ workspace, dependencies, options }) {
+
+    const zipName = `${options.configuration.archiveName || options.projectNamespace.replace(/\//g, '')}.zip`;
+    const prefixPath = `/resources/${options.projectNamespace}/`
+    const allResources = await workspace.byGlob(`${prefixPath}/**`);
+
+    const zip = new yazl.ZipFile();
+    await Promise.all(allResources.map((resource) =>
+        resource.getBuffer().then((buffer) => {
+            options.configuration && options.configuration.debug && log.info("Transpiling file " + resource.getPath());
```

Comment:
not meaning to sound kleinkariert, but is this really "Transpiling"? (assuming: "Packing" or "Adding to archive" or "Zipping" :) )